tests/testthat/test_20-coupled.R

```diff
@@ -261,17 +263,20 @@ for (csvfile in csvfiles) {
       getLine = function() stop("getLine should not called."),
       .package = "gms"
     )
-    unlink("../../*TESTTHAT.RData")
+    unlink("../../*TESTTHAT.RData", force = TRUE)
     expect_no_warning(config <- readCheckScenarioConfig(csvfile, remindPath <- file.path("..", "..")))
   })
 }
 
 test_that("delete files to leave clean state", {
   # leave clean state
   skipIfPreviousFailed()
-  if (! all(file.exists(deleteallfiles))) {
+  missingfiles <- setdiff(deleteallfiles[! file.exists(deleteallfiles)],
```

Comment:
```suggestion
  # Exclude NDC folders since they were deleted earlier to see if everything fails correctly if mag iteration is bigger than rem iteration.
  missingfiles <- setdiff(deleteallfiles[! file.exists(deleteallfiles)],
```